src/seedu/addressbook/data/person/AddressProperty.java

```diff
@@ -0,0 +1,26 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Created by raynoldng on 5/2/18.
+ */
+public class AddressProperty {
```

Comment:
This class and its non-trivial methods should have header comments. 
If not, developers who write client code for this class will not know how to use it.